### ISSUE TITLE
Can I add options to avoid loading the entire file

### ISSUE THREAD

**Issue by adminChina** [mattiasw/ExifReader#215]
HI Mattias！
   Whether it can be displayed in ```ExifReader.load``` adds an option to control how much data the network request receives to save the traffic of the request.

**Comment by mattiasw** [comment]
Hi! It wasn't possible but now it is! :-) Just released that functionality as version 4.9.0. That was a great idea, thanks! I think other users will be happy too because reading only part of the file is a recurring request.

To use it, add `length` as an option like this:
```
const tags = await ExifReader.load(filename, {length: 128 * 1024});
```

**Comment by tusamni** [comment]
Thanks for adding this, it helped speed up load times on my AstroJS site during dev. 👍

**Comment by elstoc** [comment]
Hi, I just upgraded to v4.9.0 to test this (with TypeScript) but my IDE doesn't pick up the new option.

**Comment by mattiasw** [comment]
The type definitions usually rely on pull requests. I seldom update them myself because I'm simply not very good at it. That being said, does the following change look OK? I saw another option was also missing so I added that one too.

https://github.com/mattiasw/ExifReader/pull/217

**Comment by elstoc** [comment]
> That being said, does the following change look OK?

I'm still fairly new to TypeScript but those changes look good to me.

**Comment by mattiasw** [comment]
Released as version 4.9.1.
